src/tests/test_config_functional.py

```diff
@@ -0,0 +1,39 @@
+"""
+Various tests for config
+"""
+
+import os
+import unittest
+import tempfile
+
+from .test_process import TestProcessProto
```

Comment:
remove

Reply:
`TestProcessProto` is required for `TestProcessConfig` class

Reply:
Sorry I was wrong. Please put the stop process / kill process back. Also, rename this file to `test_config_process.py`. That should be enough.